OKTA-804418 Deprecate Apps API: update /apps API links to Redocly (#5092)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md

layout: Guides
---

> This guide explains how to do the following:

Reviewer: ```suggestion
This guide explains how to complete the following steps:
```

> * Upload your Security Assertion Markup Language (SAML) certificates to sign the assertion for outbound SAML apps

Reviewer: ```suggestion
* Upload your Security Assertion Markup Language (SAML) certificates to sign the assertion for outbound SAML apps.
```

> * Sign the AuthN request

Reviewer: ```suggestion
* Sign the AuthN request.
```

> * Decrypt the assertion for inbound SAML

Reviewer: ```suggestion
* Decrypt the assertion for inbound SAML.
```

Update the key credential for the app or IdP to specify the new signing Key ID.

> * For outbound SAML, call the [Update Application API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication). Pass the app ID that you obtained in the [List your apps](#list-your-apps) step in the URL. In the body, include the app name and the app label that you obtained when you listed your apps and the Key ID that you obtained in the [Sign the CSR](#sign-the-csr) step.

Reviewer: ```suggestion
* For outbound SAML, call the [Replace an app endpoint](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication). Pass the app ID that you obtained in the [List your apps](#list-your-apps) step in the URL. In the body, include the app name and the app label that you obtained when you listed your apps and the Key ID that you obtained in the [Sign the CSR](#sign-the-csr) step.
```
